doc/changelog.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+Changes in Version 4.8
```

Comment:
The next version is 4.7 which already has an entry below.

Reply:
Lol, fixed.